ADR/ADR015-use-paper-trail-for-draft-live-or-some-other-solution.md

```diff
@@ -0,0 +1,57 @@
+# ADR015: Use Papertrail for draft/live or some other solution
+
+Date: 2023-03-03
+
+## Status
+
+Rejected
+
+## Context
+
+There is need to create live versions of a form. A form could potentially have multiple "live" forms during its lifetime
+but only one live form should be accessible at any given time. Using [PaperTrail](https://github.com/paper-trail-gem/paper_trail) 
+already creates an audit record of a form/pages record and stores the object in its previous state. 
+
+PaperTrail does have a 3rd party gem called [paper_trail-associations_tracking(PT-AT)](https://github.com/westonganger/paper_trail-association_tracking)
+that deals with [associations](https://github.com/paper-trail-gem/paper_trail#4b-associations).
+This gem should be able to give us the  functionality we needed such as trying to identify which versions of pages make 
+up a specific version of the form.
+
+### Positives
+- PT-AT can identify which forms/pages where created/updated and it was possible to restore that use to reuse
+- PT allows custom events so a new event for "make live" could be used to identify specific live versions.
+- Keep audit of events and records together in one table.
+
+### Negatives
+- PT-AT doesn't work well when deleting associations such as pages and it was found that .length/size/count after delete
+  didn't return the expected number of pages df a form as it seemed to query the pages table rather than version
+- PT-AT contributor recommends not using PT-AT because of it dependency on PT and its "blackbox" approach which doesn't seem
+  right to just assume everything works.
+
+## Decision
```

Comment:
Some of what's in here might work better in the Context section, so that the Decision itself is a bit clearer. Will make some separate comments to show what I mean

Reply:
I guess I'm thinking this decision section would just be along the lines of 'The team decided not to use PT-AT or active_snapshot to implement draft/live versions' if that makes sense,so that it's just the decision itself that's being described.